url_handlers.go

```diff
@@ -96,6 +96,24 @@ func WaitListHandler(w http.ResponseWriter, r *http.Request) {
   RenderTemplate(w, "assets/templates/waitlist.html.tmpl", nil)
 }
 
+func AssignedBuzzerHandler(w http.ResponseWriter, r *http.Request) {
+    if !IsUserLoggedIn(GetSession(w, r)) {
+      http.Redirect(w, r, "/login", 302)
+      return
+    }
+
+    if r.Method == "POST" {
+        active_party_id = r.FormValue("active_party_id")
```

Comment:
The JSON data will be in the body, not as part of a form.

See https://github.com/jfeiber/buzzer/blob/jf/sample_json_endpoint/url_handlers.go `SampleJSONHandler` for how to get data out of a request body